doc/sphinx/source/conf.py

```diff
@@ -0,0 +1,38 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+import os
+import sys
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Neo4j Graph Data Science Python Client"
+copyright = "2023, Neo4j"
+author = "Neo4j"
+release = "2023"
+
```

Comment:
seeing the options. How do we version this?
Likely we should have the same as the client.

If there are some newer endpoints (new server version), we should update this listing and ideally say for which version they are available :thinking: 

Thinking further - major server version will lead to a server major client release to make these docs valid.
We should talk about this aside from this PR on what we want to do here. Also regarding generating the algo endpoints (not only the docs).